src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -260,6 +260,10 @@
           <CopyToOutputDirectory>Always</CopyToOutputDirectory>
           <IncludeInVSIX>true</IncludeInVSIX>
         </Content>
+        <Content Include="..\..\..\NOTICE.txt">
```

Comment:
Use $(RepositoryRootDirectory) instead of the parent directory separator.

Reply:
Thanks! I'll do that

Reply:
You should probably rename it to third party notices
like 
https://github.com/dotnet/cli/blob/master/THIRD-PARTY-NOTICES

I'd add the txt at the end as it's consistent with our license.txt